xapian-bindings/python3/Makefile.am

```diff
@@ -30,51 +30,78 @@ RST_DOCS = \
 # Therefore, we don't run it by default currently.
 # replicationtest.py
 
-BUILT_SOURCES = xapian_wrap.cc xapian_wrap.h xapian.py
+BUILT_SOURCES = xapian_wrap.cc xapian_wrap.h xapian.py \
+	xapianletor_wrap.cc \
+	xapianletor_wrap.h \
+	xapianletor.py
 
-EXTRA_DIST = python.i util.i extra.i extracomments.i except.i \
+EXTRA_DIST = python.i pythonletor.i util.i utilletor.i extra.i extracomments.i except.i \
+	exceptletor.i \
 	doxy2swig.py \
 	fixup-swig-py3-wrapper \
 	generate-python-exceptions \
+	generate-letor-python-exceptions \
 	run-python-test \
 	sphinx-build \
 	testsuite.py \
 	test_xapian_star.py \
+	test_xapian_letor_star.py \
 	$(TESTS) $(BUILT_SOURCES) $(RST_DOCS)
 
 pkgpylibdir = @PYTHON3_LIB@/xapian
 
+letorpkgpylibdir = @PYTHON3_LIB@/xapianletor
+
 # Install as _DATA rather than _SCRIPTS because we don't want to make these
 # executable (they don't have a #! line).
 pkgpylib_DATA = xapian/__init__.py \
 		xapian/__pycache__/__init__.@PYTHON3_CACHE_TAG@.pyc \
 		xapian/__pycache__/__init__.@PYTHON3_CACHE_OPT1_EXT@
 
+# Install as _DATA rather than _SCRIPTS because we don't want to make these
+# executable (they don't have a #! line).
+letorpkgpylib_DATA = xapianletor/__init__.py \
+		xapianletor/__pycache__/__init__.@PYTHON3_CACHE_TAG@.pyc \
+		xapianletor/__pycache__/__init__.@PYTHON3_CACHE_OPT1_EXT@
+
 pkgpylib_LTLIBRARIES = _xapian.la
 
+letorpkgpylib_LTLIBRARIES = _xapianletor.la
 # Remove the .la file - _xapian.la is never linked against (it's a module)
 # and Python doesn't use libltdl.  Note that the library gets installed by
 # install-data, so that's where we need to hook.
 install-data-hook:
 	rm -f $(DESTDIR)$(pkgpylibdir)/_xapian.la
+	rm -f $(DESTDIR)$(letorpkgpylibdir)/_xapianletor.la
 
 # Because we don't install the .la file, "make uninstall" doesn't work and
 # we need to remove the file ourselves.
 uninstall-local:
 	rm -f $(DESTDIR)$(pkgpylibdir)/_xapian$(PYTHON3_SO)
+	rm -f $(DESTDIR)$(letorpkgpylibdir)/_xapianletor$(PYTHON3_SO)
 
 AM_CPPFLAGS = -I$(PYTHON3_INC)
-AM_CXXFLAGS = $(SWIG_CXXFLAGS) $(XAPIAN_CXXFLAGS)
+AM_CXXFLAGS = $(SWIG_CXXFLAGS) $(XAPIAN_CXXFLAGS) $(XAPIANLETOR_CXXFLAGS)
```

Comment:
I think this now should be target-specific, meaning that both `XAPIAN_CXXFLAGS` and `XAPIANLETOR_CXXFLAGS` move out of `AM_CXXFLAGS` into [per-object variables](https://www.gnu.org/software/automake/manual/html_node/Per_002dObject-Flags.html). I _think_ they belong in `xapianletor_la_CXXFLAGS`. I think also that `SWIG_CXXFLAGS` belongs with them, because the `AM_CXXFLAGS` [won't be included automatically](https://www.gnu.org/software/automake/manual/html_node/Program-and-Library-Variables.html#index-per_002dtarget-compilation-flags_002c-defined).

(@ojwb should jump in if I've got the automake details wrong!)

Reply:
fixed in a fixup commit.